File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int result = 0;
        int temp = 0;
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] == 1) {
                temp++;
                if(temp > result) {
                    result = temp;
                }
            } else {
                if(temp > result) {
                    result = temp;
                }
                temp = 0;
            }
        }

        return result;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let arr = []
        let result = false
        for(let i = 0; i<nums.length; i++) {
            const x = nums[i]
            if(arr.includes(x)) {
                result = true
            } else {
                arr.push(x)
            }
        }

        return result
    }
}
